scipy/spatial/transform/rotation.py

```diff
@@ -281,3 +321,105 @@ def as_rotvec(self):
             return rotvec[0]
         else:
             return rotvec
+
+    @classmethod
+    def from_euler(cls, seq, angles, degrees=False):
+        """Initialize rotation from Euler angles.
+
+        Parameters
+        ----------
+        seq : string
+            Up to 3 characters belonging to the set {'X', 'Y', 'Z'} for
+            intrinsic rotations [1]_, or {'x', 'y', 'z'} for extrinsic
+            rotations [2]_. Extrinsic and intrinsic rotations cannot be mixed
+            in one function call.
+        angles : float or array_like, shape (N, ) or (N, [1 or 2 or 3])
+            Euler angles specified in radians (`degrees` is False) or degrees
+            (`degrees` is True).
+            For a single character `seq`, `angles` can be:
+
+                - a single value
+                - array_like with shape (N, ), where each `angle[i]`
+                  corresponds to a single rotation
+                - array_like with shape (N, 1), where each `angle[i, 0]`
+                  corresponds to a single rotation
+
+            For 2- and 3-character wide `seq`, `angles` can be:
+
+                - array_like with shape (W, ) where `W` is the width of
+                  `seq`, which corresponds to a single rotation with `W` axes
+                - array_like with shape (N, W) where each `angle[i]`
+                  corresponds to a sequence of Euler angles describing a single
+                  rotation
+
+        degrees : boolean, optional
+            If True, then the given angles are taken to be in degrees. Default
+            is False.
+
+        References
+        ----------
+        .. [1] `Intrinsic rotations
```

Comment:
Can we combine the two references together? It will be more logical and concise.